### PR TITLE
Event

### PR DESCRIPTION
**Pull request by davidrkupton**
Adds event content type.

### CODE REVIEW

File: docroot/modules/custom/bos_content/modules/node_event/node_event.module [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  // Render the downloads (field_links) into a details_item.
+  if (isset($variables["content"]["field_links"])) {
+    $vars["label"] = $variables["content"]["field_links"]["#title"] . ":";
+    $vars["body"] =  $variables["content"]["field_links"];
+    $variables["content"]["field_links"] = Drupal::theme()
+      ->render("detail_item", $vars);
+  }
+
+  if (!empty($variables["content"]["field_city_hall"]) ) {
+    // TODO: Ajax back to get the address for CityHall.
```
Comment (jimafisk) [inline docroot/modules/custom/bos_content/modules/node_event/node_event.module]: I assume this is ok coming back to later?

File: docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig [hunk 2]
```diff
@@ -36,80 +36,80 @@
...
-            <?php if (!empty($node) && ($node->type !== 'tabbed_content' && $node->type !== 'how_to')): ?>
-            <?php if ($node->type !== 'topic_page'): ?>
-            <style>
-                .hro {
-                    background-image: url(<?php print $small_image ?>);
-                }
+    <div class="main">
+        <div class="container">
+            <section class="main-content" id="content" role="main">
+                <?php print render($page['highlighted']); ?>
```
Comment (jimafisk) [inline docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig]: Looks like this still needs to be converted to twig

File: docroot/themes/custom/bos_theme/templates/layout/page.html.twig [hunk 3]
```diff
@@ -97,9 +102,10 @@
           <ul class="action-links">{{ action_links }}</ul>
         {%  endif %}
 
-        {% if header_image %}
-          {% if node and not node.type == 'tabbed_content' and not node.type == 'how_to' %}
-            {% if not node.type == 'topic_page' %}
+        {% if node and header_image %}
+          {% if node.type.entity.label|lower != 'tabbed_content' and node.type.entity.label|lower != 'how_to' %}
+            {% if node.type.entity.label|lower != 'topic_page' %}
+              {{ breakpoint() }}
```
Comment (jimafisk) [inline docroot/themes/custom/bos_theme/templates/layout/page.html.twig]: twig_debug breakpoint can be removed when dev is finished

**Comment by davidrkupton** [comment]
link #243

**Pull request merged**